stories/module-analytics.stories.js

```diff
@@ -124,6 +127,42 @@ storiesOf( 'Analytics Module', module )
 			</WithTestRegistry>
 		);
 	} )
+	.add( 'Property Select including GA4 properties', () => {
+		const { accounts, properties, profiles } = fixtures.accountsPropertiesProfiles;
+		/* eslint-disable sitekit/acronym-case */
+		const accountID = properties[ 0 ].accountId;
+		const propertyID = profiles[ 0 ].webPropertyId;
+		/* eslint-enable */
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( STORE_NAME ).receiveGetAccounts( accounts );
+			dispatch( STORE_NAME ).finishResolution( 'getAccounts', [] );
+
+			// eslint-disable-next-line sitekit/acronym-case
+			dispatch( STORE_NAME ).receiveGetProperties( properties, { accountID: properties[ 0 ].accountId } );
+			dispatch( STORE_NAME ).receiveGetProfiles( profiles, {
+				accountID,
+				propertyID,
+			} );
```

Comment:
![need-profiles-for-each-option](https://user-images.githubusercontent.com/8496063/119138217-f15d5b80-ba41-11eb-9893-8c6feeb9a6e7.png)

How concerned are you about this @eugene-manuilov ?  

We already have console errors on storybook which, while not ideal, doesn't break anything

Like on the other story on the other PR, I need to add profiles for each `account` and `property` ID

Reply:
Yeah, we don't need to make it fully functional and recreate the whole environment. That error appears only when you select something. So we can ignore it.